### PR TITLE
Accessibility UI fixes

### PR DESCRIPTION
**Pull request by alfogrillo**
Fix accessibility issues on:
- Attachment picker
- RTE toolbar

| **Before** | **After** |
| --- | --- |
|b1|a1|
|b2|a2|

### CODE REVIEW

File: UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.userSessionScreenRTE-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.userSessionScreenRTE-1.png:1]: Was it intentional to increase the spacing here? I'm not sure I see *how* that happened from looking at the code changes on GH 🤔

Comment (alfogrillo) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.userSessionScreenRTE-1.png:1]: Do you mean now there is less vertical space in the attachment menu?
I think this is related to [this change](https://github.com/vector-im/element-x-ios/pull/1685/files#diff-64576de4e4aa60e3a71267502830fff46481d5681dc87857f917384c4294afc4R121).
Icons used to be 24pt but now with the `.resizable()` modifier Images fill all the available space. 
I needed to adjust them to 20pt as for the design.

Comment (pixlwave) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.userSessionScreenRTE-1.png:1]: My bad, I was looking at the swipe the wrong way around 🤦‍♂️

**Review by pixlwave — APPROVED** [review]
LGTM 👍

**Pull request merged**